app/importers/iris_record_importer.rb

```diff
@@ -30,12 +30,18 @@ def create_for(record:)
                      "#{record.respond_to?(:title) ? record.title : record}."
 
       work_type = record.resource_type.first.classify.constantize
-
       attributes = record.attributes
+      # TODO: possible refactor as more info from UCSB becomes available, 'shapefile' might not be necessary to distinguish from 'binary_files' here, but for now it's clear.
       binary_file = attributes.delete(:representative_file)
+      shape_files = attributes.delete(:shapefiles)
+
+      # geo_mime_type tells geo_works how to process the binaries
+      geo_mime_type = attributes.delete(:geo_mime_type)
```

Comment:
Is it okay to assume that all the attached files have the same mime type?

Reply:
They definitely won't, there's a set of specific types that geo_works handles.